readthedocs/templates/core/project_bar_base.html

```diff
@@ -45,6 +45,8 @@ <h1>
 
         <li class="{{ builds_active }}"><a href="{{ project.get_builds_url }}">{% trans "Builds" %}</a></li>
 
+        <li class="{{ pr_builds_active }}"><a href="{{ project.get_pr_builds_url }}">{% trans "PR Builds" %}</a></li>
```

Comment:
This should only show up if the user _has_ any PR builders.

Reply:
@ericholscher Updated the PR